activities/MazeWeb.activity/js/activity.js

```diff
@@ -53,6 +53,24 @@ define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar
             }
         });
 
+        document.getElementById("fullscreen-button").addEventListener('click', function() {
+            document.getElementById("main-toolbar").style.opacity = 0;
+            console.log("1");
+            document.getElementById("canvas").style.top = "0px";
+            console.log("2");
+            document.getElementById("unfullscreen-button").style.visibility = "visible";
+            console.log("3");
+            onWindowResize();
+            console.log("4");
+        });
+        
+		document.getElementById("unfullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 1;
+			document.getElementById("canvas").style.top = "55px";
+			document.getElementById("unfullscreen-button").style.visibility = "hidden";
+			onWindowResize();
+		});
+
 		var generateXOLogoWithColor = function(color) {
 			var coloredLogo = xoLogo;
 			coloredLogo = coloredLogo.replace("#010101", color.stroke)
```

Comment:
Indentation here is out of line.

Reply:
The generateXOLogoWithColor needs to move back as well.

Reply:
All set! Thank you.